tests/udp_rendezvous_server.rs

```diff
@@ -0,0 +1,56 @@
+extern crate maidsafe_utilities;
+extern crate mio;
+extern crate mio_extras;
+extern crate p2p;
+extern crate rust_sodium as sodium;
+extern crate serde_json;
+#[macro_use]
+extern crate unwrap;
+extern crate net2;
+extern crate socket_collection;
+
+mod utils;
+
+use mio::Poll;
+use p2p::{Config, Interface, NatMsg, Res, UdpRendezvousClient, UdpRendezvousServer};
+use socket_collection::UdpSock;
+use std::net::SocketAddr;
+use std::sync::mpsc;
+use utils::spawn_event_loop;
+
+#[test]
+fn it_responds_with_client_address() {
+    let server_sock = unwrap!(UdpSock::bind(&unwrap!("127.0.0.1:0".parse())));
```

Comment:
Check other review comments for reference. Having a config to configure the library and then designing the library to ignore that config doesn't seem nice and especially so if it's only needed by the test. Pls use in-mem config file if needed as explained in the prior review comment.